src/Bramus/Router/Router.php

```diff
@@ -358,9 +358,9 @@ private function handle($routes, $quitAfterRun = false)
                 }, $matches, array_keys($matches));
 
                 // Call the handling function with the URL parameters if the desired input is callable
-                $this->invoke($route['fn'], $params);
-
-                ++$numHandled;
+                if (true === $this->invoke($route['fn'], $params)) {
```

Comment:
I'm not sure why you adding this check to `numHandled`. `numHandled` isn't a counter of successful callbacks, but yes successful iterations of the router.